ansible/roles/ami-ovf/tasks/main.yml

```diff
@@ -1,4 +1,23 @@
 ---
+    - name: Packages                   | Remove PMM2 Testing Server repository
+      yum_repository:
+        name: pmm2-server
+        state: absent
+
+    - name: Packages                   | Clean up yum metadata
+      command: yum clean metadata
```

Comment:
Does it make sense to use `yum clean all` there for extra safety?

Reply:
I think it is ok to use 'yum clean metadata'.